src/contacts/tests/test_contacts.py

```diff
@@ -0,0 +1,331 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+Tests for contacts
+"""
+
+import random
+import unittest
+import json
+from unittest.mock import patch, mock_open
+
+from sqlalchemy.exc import SQLAlchemyError
+
+from contacts.contacts import create_app
+from contacts.tests.constants import (
+    EXAMPLE_CONTACT,
+    EXAMPLE_USER,
+    EXAMPLE_PUBLIC_KEY,
+    EXAMPLE_HEADERS,
+    EXAMPLE_USER_PAYLOAD,
+    INVALID_ACCOUNT_NUMS,
+    INVALID_LABELS,
+    INVALID_ROUTING_NUMS,
+)
+
+def create_new_contact(**kwargs):
+    """Helper method for creating new contacts from template"""
+    example_contact = EXAMPLE_CONTACT.copy()
+    example_contact.update(kwargs)
```

Comment:
this just looks for key/values passed in and updates them right?

Reply:
yeah so if there are any arguments passed we can unpack them using `**` to a dict. Conveniently, dicts can be updated using another dict so it lets us override any default value we have set in the example.